classes/TotFormat.php

```diff
@@ -124,7 +124,7 @@ public static function formatDescription($desc)
             $desc = preg_replace('/<script\b[^>]*>(.*?)<\/script>/is', "", $desc);      
             $desc = preg_replace('/ on([A-Za-z0-9]*)="(.*)"/is', "", $desc);
             
-        }
```

Comment:
It appears that what we wanted to do (deleting javascript was problematic for some users. Indeed, eBay does accept javascript in the description, only it has some rules that we cannot predict.
It is better to let the user decide and make some tests to check if the description will be allowed.